src/Servers/IIS/AspNetCoreModuleV2/InProcessRequestHandler/inprocessapplication.cpp

```diff
@@ -251,10 +251,26 @@ IN_PROCESS_APPLICATION::ExecuteApplication()
             LOG_INFOF(L"Setting current directory to %s", this->QueryApplicationPhysicalPath().c_str());
         }
 
+        auto redirectionOutput = LoggingHelpers::CreateOutputs(
+            m_pConfig->QueryStdoutLogEnabled(),
+            m_pConfig->QueryStdoutLogFile(),
+            QueryApplicationPhysicalPath(),
+            m_stringRedirectionOutput
+        );
+
+        StandardStreamRedirection redirection(*redirectionOutput.get(), m_pHttpServer.IsCommandLineLaunch());
```

Comment:
Moving the StandardStreamRedirection up should be fine, it's now just called slightly earlier. No other methods modify the redirection.

ForwardingRedirectionOutput should also be fine, it still will be closed before StandardStreamRedirection.